prime-router/src/testIntegration/resources/datatests/mappinginventory/nte/NTE-to-annotation.fhir

```diff
@@ -85,24 +85,24 @@
           {
             "role" : "source",
             "what" : {
-              "reference" : "Device/1699911340123383000.be5a1852-0519-4647-b0da-5c51ae7b40dc"
+              "reference" : "Device/1700068094890673000.3a8d27df-99b9-454c-9892-7b41b75b9a5f"
             }
           }
         ]
       }
     },
     {
-      "fullUrl" : "Device/1699911340123383000.be5a1852-0519-4647-b0da-5c51ae7b40dc",
+      "fullUrl" : "Device/1700068094890673000.3a8d27df-99b9-454c-9892-7b41b75b9a5f",
```

Comment:
Ah I see this is already occurring here, so you can probably ignore my other comment